packages/at_secondary_server/test/update_verb_test.dart

```diff
@@ -914,6 +921,187 @@ void main() {
       }
     });
 
+    test('Concurrent updates to the same IMMUTABLE record', () async {
```

Comment:
we kick off 10 update verbs concurrently for an immutable record. Since immutable, only one of them should succeed, all of the others should fail.